functions-and-operators/aggregate-group-by-functions.md

```diff
@@ -63,6 +63,8 @@ In addition, TiDB also provides the following aggregate functions:
     1 row in set (0.00 sec)
     ```
 
+Except for the `GROUP_CONCAT()` and `APPROX_PERCENTILE()` functions, all the preceding functions can serve as [window functions](/functions-and-operators/window-functions.md).
```

Comment:
```suggestion
Except for the `GROUP_CONCAT()` and `APPROX_PERCENTILE()` functions, all the preceding functions can serve as [Window functions](/functions-and-operators/window-functions.md).
```
https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings